components/org.wso2.carbon.identity.organization.management.service/src/main/java/org/wso2/carbon/identity/organization/management/service/OrganizationManagerImpl.java

```diff
@@ -842,16 +842,12 @@ private void createTenant(String domain, String orgCreatorID, String orgCreatorN
         }
     }
 
-    private Tenant createTenantInfoBean(String domain, String orgCreatorID) {
+    private Tenant createTenantInfoBean(String domain, String orgCreatorID, String orgCreatorName) {
 
         Tenant tenant = new Tenant();
         tenant.setActive(true);
         tenant.setDomain(domain);
-        /*
-        Set the creator's UUID as the tenant admin name because tenant data model doesn't store the admin uuid,
-        and it is required when creating the org-user association.
-         */
-        tenant.setAdminName(orgCreatorID);
+        tenant.setAdminName(orgCreatorName);
         tenant.setAdminUserId(orgCreatorID);
         tenant.setEmail("dummyadmin@email.com");
         // set the password as domain for now to avoid findbugs detecting it as a hardcoded value.
```

Comment:
If we avoid setting anything for email and password, will it break anything?

Reply:
There is an email non-empty validation https://github.com/wso2/carbon-multitenancy/blob/5a9360e09fcc4b0dfa3e036626c99b3d598c1be4/components/tenant-mgt/org.wso2.carbon.tenant.mgt/src/main/java/org/wso2/carbon/tenant/mgt/services/TenantMgtImpl.java#L396

Reply:
password is not mandatory. So I'll remove that